### PR TITLE
fix opcode overwrite issue in fast interp

### PR DESCRIPTION
**Pull request by xujuntwt95329**

### CODE REVIEW

**Review by yamt — COMMENTED** [review]
mini loader seems to have the same bug.

File: core/iwasm/interpreter/wasm_loader.c [hunk 1]
```diff
@@ -8064,6 +8064,9 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
             case WASM_OP_SELECT_T:
             {
                 uint8 vec_len, ref_type;
+#if WASM_ENABLE_FAST_INTERP != 0
+                uint8 *p_code_compiled_orig = loader_ctx->p_code_compiled;
```
[the remark this answers is not in the document]
Comment (xujuntwt95329) [reply core/iwasm/interpreter/wasm_loader.c]: Thanks, updated

**Comment by xujuntwt95329** [comment]
> mini loader seems to have the same bug.

Yes, updated. Thanks for reminding.

**Review by wenyongh — APPROVED** [review]
LGTM

**Review by yamt — APPROVED** [review]
lgtm

**Comment by erxiaozhou** [comment]
Nice job

**Pull request merged**